.github/workflows/label.yml

```diff
@@ -0,0 +1,24 @@
+name: Labeller
+on:
+  pull_request_target:
+    types: [closed]
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: .github/scripts
+          ref: main
+      - name: Add release labels on merge
+        run: |
+          .github/scripts/pr-label-issues.sh "${{ github.event.pull_request.number }}" "$GITHUB_REPOSITORY" "$GITHUB_BASE_REF" >> "$GITHUB_STEP_SUMMARY"
```

Comment:
you need to add that script to this commit

Reply:
> you need to add that script to this commit

I changed the sparse check out to get the files always from main, so no need to duplicate the script